test/Recipe-test.js

```diff
@@ -0,0 +1,14 @@
+import { expect } from 'chai';
```

Comment:
TDD- going to start here as we keep going. Did want to put in the file for the class and do the bare bones outline.